src/Components/Music Player/musicPlayer.tsx

```diff
@@ -0,0 +1,35 @@
+import styles from './musicPlayer.module.css';
+
+type MusicProps = {
+  title: string;
+  url: string;
+};
+
+function MusicPlayer({ title, url }: MusicProps): JSX.Element {
+  const audio = new Audio(url);
+  let content;
+
+  function handleClick() {
+    if (audio.paused) {
+      audio.play();
+      content = '⏹';
+    } else {
+      audio.pause();
+      content = '▶️';
+    }
+  }
```

Comment:
This is probably not going to work correctly. You need a state for your content. 
That will change your button content, but then you'll have the problem that your audio will be rendered again and again every time that your state changes. For that, you could use [useMemo](https://reactjs.org/docs/hooks-reference.html#usememo).

```suggestion
  const audio = new Audio(url);
 const [content, setContent] = useState('▶️');

  function handleClick() {
    if (audio.paused) {
      audio.play();
      setContent('⏹');
    } else {
      audio.pause();
      setContent('▶️');
    }
  }
```